index.html

```diff
@@ -118,6 +118,20 @@
       </pre>
       </div>
     </div>
+    
+    <div id="7-Technical-Sins" class="step slide" data-x="3000" data-y="0">
+      <q><b>Technical Sins :</b></q> <sup>[4a]</sup>

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this <b> tag by <strong>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)